palindrom

```diff
@@ -0,0 +1,34 @@
+function pal(start , end , thisword){
+
+   
+    if( thisword[start] != thisword[end] ){
+        console.log(thisword[start]);
```

Comment:
Debugging code (such as `console.log`) should not be included in the final implementation, only while development.